src/gobapi/profiled_query.py

```diff
@@ -0,0 +1,62 @@
+import time
+
+from sqlalchemy import event
+from sqlalchemy.engine import Engine
+
+
+class ProfiledQuery():
+
+    LONG_DURATION = 60 * 10  # 10 minutes or longer
```

Comment:
Maybe start with a slightly higher value and work our way down? Say 30 minutes? Many queries take 10 minutes at the moment.

Reply:
Set to 30